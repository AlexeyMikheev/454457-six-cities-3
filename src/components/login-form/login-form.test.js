import React from "react";
import Enzyme, {mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import LoginForm from "./login-form.jsx";

Enzyme.configure({
  adapter: new Adapter(),
});

it(`Render Login`, () => {
  const div = document.createElement(`div`);
  document.body.appendChild(div);

  const loginFormComponent = mount(<LoginForm onValueChanged={jest.fn()} onSubmitForm={jest.fn()}/>, {attachTo: div});

  expect(loginFormComponent.getDOMNode()).toMatchSnapshot();
});
