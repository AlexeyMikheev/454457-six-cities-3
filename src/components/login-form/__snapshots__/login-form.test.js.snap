// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Login 1`] = `
<form
  action=""
  className="login__form form"
  onSubmit={[Function]}
>
  <div
    className="login__input-wrapper form__input-wrapper"
  >
    <label
      className="visually-hidden"
      htmlFor="email"
    >
      E-mail
    </label>
    <input
      className="login__input form__input"
      id="email"
      name="email"
      placeholder="Email"
      type="email"
    />
  </div>
  <div
    className="login__input-wrapper form__input-wrapper"
  >
    <label
      className="visually-hidden"
      htmlFor="password"
    >
      Password
    </label>
    <input
      className="login__input form__input"
      id="password"
      name="password"
      placeholder="Password"
      type="password"
    />
  </div>
  <button
    className="login__submit form__submit button"
    type="submit"
  >
    Sign in
  </button>
</form>
`;
